src/Components/ArticleList.jsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from "react"
+import { Card, CardHeader } from "@mui/material";
+
+function ArticleList(){
+    const [articlesList, setArticlesList] = useState([]);
+    const myHeaders = new Headers();
+    myHeaders.append("Accept", "application/json");
+
+    const fetchArticles = () =>{
+        const apiPath = 'https://fiadh-nc-news.onrender.com/api/articles'
+        const requestOptions = {
+            method: "GET",
+            headers: myHeaders,
+            redirect: "follow"
+        }
+
+        fetch(apiPath, requestOptions)
+        .then((response)=>response.json())
+        .then(({articles})=>{
+            setArticlesList(articles)
+            console.log(articles)
+        })
+        .catch((err)=>{
+            console.log(err)
+        })
+    }
+
+    useEffect(()=>{
+        fetchArticles();
```

Comment:
Your "setArticlesList" function is happening inside your "fetchArticles", which works, but you may need to re-use that function later to set other things. Consider doing that outside of the "fetchArticles" function.